server/shared/database/migrations/20190613113029-remove-user-token-field.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const TABLE = 'user';
+const COLUMN = 'token';
+const options = { type: Sequelize.STRING, unique: true };
+
+module.exports = {
```

Comment:
Rather then removing it should we use it upon reset for invalidation?

Reply:
I don't get you; what do you mean by: _upon reset for invalidation_ 🤔 This system relies on _auto_-invalidation of reset tokens because their secret is generated using previous password hash.

Reply:
Missed that change 👍🏻

Reply:
Make sure INTEGRATION user is handled since this will obviously invalidate tokens for preview. Set proper warnings. We should have drafts for breaking changes / release notes so we don't forget this.

Reply:
Tbh I totally forgot about integration users but we should be able to handle them differently by checking their audience maybe?